packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -189,9 +189,11 @@ export interface IPluginLedgerConnectorFabricOptions
   cliContainerGoPath?: string;
   cliContainerEnv: NodeJS.ProcessEnv;
   pluginRegistry: PluginRegistry;
-  sshConfig: SshConfig;
+  sshConfig?: SshConfig;
+  sshConfigBase64Encoded?: string;
   readonly sshDebugOn?: boolean;
-  connectionProfile: ConnectionProfile;
+  connectionProfile?: ConnectionProfile;
+  connectionProfileBase64Encoded?: string;
```

Comment:
@jagpreetsinghsasan I recommend calling it `connectionProfileB64` or `connectionProfileBase64` to be more concise but still carry the same information